rest/src/routes/networkRoutes.js

```diff
@@ -69,10 +71,16 @@ module.exports = {
 		server.get('/network/fees/transaction', (req, res, next) => {
 			const numBlocksTransactionFeeStats = services.config.numBlocksTransactionFeeStats || 1;
 			const latestBlocksFeeMultiplier = db.latestBlocksFeeMultiplier(numBlocksTransactionFeeStats);
-			return Promise.all([readAndParseNodePropertiesFile(), latestBlocksFeeMultiplier]).then(feeMultipliers => {
+			return Promise.all([readAndParseNodePropertiesFile(), latestBlocksFeeMultiplier,
+				readAndParseNetworkPropertiesFile()]).then(feeMultipliers => {
+				// defaultDynamicFeeMultiplier -> uint32
+				const defaultDynamicFeeMultiplier = parseInt(sanitizeInput(
+					feeMultipliers[2].chain.defaultDynamicFeeMultiplier
+				), 10);
+				const defaultedFeeMultipliers = feeMultipliers[1].map(f => (0 === f ? defaultDynamicFeeMultiplier : f));
 				res.send({
-					averageFeeMultiplier: Math.floor(average(feeMultipliers[1])),
-					medianFeeMultiplier: Math.floor(median(feeMultipliers[1])),
+					averageFeeMultiplier: Math.floor(average(defaultedFeeMultipliers)),
+					medianFeeMultiplier: Math.floor(median(defaultedFeeMultipliers)),
 					highestFeeMultiplier: Math.max(...feeMultipliers[1]),
 					lowestFeeMultiplier: Math.min(...feeMultipliers[1]),
 					minFeeMultiplier: Number(feeMultipliers[0].node.minFeeMultiplier.replace('\'', ''))
```

Comment:
should it be sanitizeInput(feeMultipliers[0].node.minFeeMultiplier)

Reply:
don't think so, it is already number